src/train.py

```diff
@@ -1,50 +1,71 @@
 import argparse
 from training_data import TrainingData
 from rasa_nlu.util import update_config
-import json
+import json, warnings, os
 
 
 def create_argparser():
     parser = argparse.ArgumentParser(description='train a custom language parser')
+
     # TODO add args for training only entity extractor or only intent
     parser.add_argument('-b', '--backend', default=None, choices=['mitie', 'spacy_sklearn'],
                         help='which backend to use to interpret text (default: None i.e. use built in keyword matcher).')
     parser.add_argument('-p', '--path', default=None, help="path where model files will be saved")
     parser.add_argument('-d', '--data', default=None, help="file containing training data")
     parser.add_argument('-c', '--config', required=True, help="config file")
     parser.add_argument('-l', '--language', default='en', choices=['de', 'en'], help="model and data language")
+    parser.add_argument('-m','--mitie_file', default='data/total_word_feature_extractor.dat', help='file with mitie total_word_feature_extractor')    
```

Comment:
have flattened the configuration structure, so it's no longer
```
{"backends" : 
  "mitie": { "fe_file": "/path/to"}
}
```
but just `"mitie_file"` at the top level 
this makes it simpler to set via environment variable.